Lab#2.ipynb

```diff
@@ -0,0 +1,722 @@
+      "text/plain": [
+       "<Figure size 432x288 with 1 Axes>"
+      ]
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "n = 10\n",
+    "sumpmf = pmf\n",
+    "for i in range(n-1):\n",
+    "    sumpmf = np.convolve(sumpmf, pmf)\n",
+    "\n",
+    "x = np.arange(xbase[0]*n, xbase[-1]*n+1)\n",
+    "fig = plt.bar(x, sumpmf, width=1)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Then we would have seen $16 \\times 10 = 160$ gamma rays. Find the index of x corresponding to 140 rays, and the same index in our probability mass function would correspond to the probability of seeing 160 rays over the time period. Summing the probabilities until that point gives us (1 - P) where P is the probability that the background could produce 160 or more cosmic rays."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 8,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Probability that background could have produced 160 rays or \n",
+      "more over 10 days : 0.0000008904%\n",
+      "Sigma Certainty that we observed a gamma ray source : 5.632σ\n"
+     ]
+    }
+   ],
+   "source": [
+    "indx = list(x).index(160)\n",
+    "P = 1 - sum(sumpmf[:indx+1])\n",
+    "print(f\"Probability that background could have produced 160 rays or \\nmore over 10 days : {P*100:.10f}%\")\n",
+    "sig = stats.norm.ppf(1-P)\n",
+    "print(f\"Sigma Certainty that we observed a gamma ray source : {sig:.3f}σ\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Problem 2"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 9,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "0.9999999166666617\n"
```

Comment:
you printed out this value for this one graph and I can't immediately tell what it means.